.buildkite/pipeline.yaml

```diff
@@ -11,8 +11,17 @@ x-defaults: &defaults
         run: build
 
 steps:
+  - name: ":sleuth_or_spy: Checking for release tag"
+    key: "is_released"
+    branches: "main"
+    # we assume that all releases with have a tag and that tag with have a @ version format
+    if: build.tag =~ /@(\d+\.\d+\.\d+)/
```

Comment:
Would canary's be caught up in this?

Reply:
That's a good point, could potentially have a negative assertion for the string "canary" and check like this:
```
build.tag =~ /@(\d+\.\d+\.\d+)/ && build.tag =~ /^(?:(?!canary).)*$/
```

Reply:
Which should work:
![Screenshot 2023-11-23 at 9 56 48 am](https://github.com/cultureamp/kaizen-design-system/assets/36558508/82367ff2-f821-4713-b020-df87e0f8c021)
![Screenshot 2023-11-23 at 9 58 28 am](https://github.com/cultureamp/kaizen-design-system/assets/36558508/cfa9c673-42b8-4ba3-86da-661104ed4ebb)

Reply:
I'd combine them together but the regex looks pretty gnarly 😆 
`(?:(?!canary)|@(\d+\.\d+\.\d+)).*$`